### PR TITLE
Gc add on

### PR DESCRIPTION
**Pull request by sluetze**
## Description
added add-on role for garbagecollection

## Checklist/ToDo's

- [ ] Added documentation?
- [ ] Added release note entry? (  docs/release-notes.md )
- [ ] Tested?

### CODE REVIEW

**Comment by sluetze** [comment]
fixed the lint errors. i obviously wasnt concentrated when building the branch and included some commits which are related to another PR. I close the other PR, since its quite small stuff

File: ansible/roles/openshift-4-cluster/tasks/post-install-add-ons.yml [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+    - post-install-add-ons
 
 - name: "Handle post_install_add_ons (include_role)"
   ansible.builtin.include_role:
     name: "{{ item.name }}"
     tasks_from: "{{ item.tasks_from | default('main.yml') }}"
+    apply:
+      tags:
+        - post-install
+        - add-ons
```
Comment (rbo) [inline ansible/roles/openshift-4-cluster/tasks/post-install-add-ons.yml:16]: Please remove `add-ons` let's just use `post-install` and `post-install-add-ons`.

File: ansible/roles/openshift-4-cluster/tasks/post-install-add-ons.yml [hunk 2]
```diff
@@ -1,11 +1,20 @@
 ---
 - name: Include vars of stuff.yaml into the 'stuff' variable (2.2).
   ansible.builtin.include_vars: "{{ playbook_dir }}/../add-ons.yml"
+  tags:
+    - post-install
+    - add-ons
```
Comment (rbo) [inline ansible/roles/openshift-4-cluster/tasks/post-install-add-ons.yml:6]: Please remove `add-ons` let's just use `post-install` and `post-install-add-ons`.

**Review by rbo — CHANGES_REQUESTED** [review]
Please remove tags `add-ons` let's just use `post-install` and `post-install-add-ons` thanks.

**Comment by rbo** [comment]
LGTM

**Comment by rbo** [comment]
Thanks for contribution.

**Pull request merged**